samples/src/main/python/SynapseSample.py

```diff
@@ -92,7 +95,8 @@
     .option("spark.synapse.linkedService", "<link service name>") \
     .option("kustoDatabase", "<Database name>") \
     .option("kustoTable", "<Table name>") \
-    .option("sparkIngestionPropertiesJson", sp.toString()) \
+   #.option("authType", "LS") \ # if this added it uses the link service credentials, default is native - that will take the logged-in user credentials or the Synapse Workspace MSI for automated excution.            
```

Comment:
```suggestion
    #.option("authType", "LS") \ # If this is added, it uses the link service credentials. Otherwise, the default is native - using the logged-in user's credentials or the Synapse Workspace managed identity for automated execution.
```

Reply:
Its a little anoying in python please make sure this compiles with the line commented out like this , i think it only compiles now if you delete the line completely

Reply:
Please close the other pr

Reply:
Which other PR?